src/entities/appointment.entity.ts

```diff
@@ -32,6 +32,12 @@ export default class Appointment {
   @Column({ nullable: false })
   duration: number;
 
+  @Column({ nullable: false })
```

Comment:
You should use the typeORM decorators for these two columns as we agreed before.
```suggestion
   @CreateDateColumn()
  created_at: Date;

  @UpdateDateColumn()
  updated_at: Date;
```